### ISSUE TITLE
Fix cftp man page syntax

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8601]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8601|
|Type|defect|
|Created|2016-07-13 19:48:09Z|
|Branch|https://github.com/twisted/twisted/tree/8601-scop-cftpman|

```
$ man --warnings docs/conch/man/cftp.1 >/dev/null
Usage: .Nm name ... (#15)
Usage: .Nm name ... (#31)
```

<details><summary>Searchable metadata</summary>

```
trac-id__8601 8601
type__defect defect
reporter__scop scop
priority__normal normal
milestone__None None
branch__8601_scop_cftpman 8601-scop-cftpman
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1468439289942159 1468439289942159
changetime__1468449627740733 1468449627740733
version__None None
owner__scop scop
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_2"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/330

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @scop|
|-|-|

Look good. thanks. Will merge!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 81b7f0a37d90e815e4c720b3e8aec91044e9a8e9
```
#!CommitTicketReference repository="" revision="81b7f0a37d90e815e4c720b3e8aec91044e9a8e9"
Merge 8601-scop-cftpman: Fix man page for cftp.

Author: scop
Reviewer: adiroiban
Fixes: #8601
```
